api-reference/10 UI Components/dxPivotGrid/1 Configuration/export/export.md

```diff
@@ -4,6 +4,8 @@ type: Object
 ---
 ---
 ##### shortDescription
+[tags] xlsx, csv
+
 Configures client-side exporting.
 
 ---
```

Comment:
Can we update line 10 (below) as follows?

`A user can click the **Export** button to save an Excel file with the exported data. Data types, sort, filter, and group settings are maintained.`